src/app/customer/customer-matches/customer-matches.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { UserService } from "../../user/user.service";
+import { ToasterService } from "../../toaster/toaster.service";
+import { MatchService } from "@boklisten/bl-connect";
+import { UserDetail, UserMatch } from "@boklisten/bl-model";
+
+@Component({
+	selector: "app-customer-matches",
+	templateUrl: "./customer-matches.component.html",
+	styleUrls: ["./customer-matches.component.scss"],
+})
+export class CustomerMatchesComponent implements OnInit {
+	@Input() customer: UserDetail;
+	public userMatchesLocked: boolean;
+	public userMatches: UserMatch[];
+
+	public isAdmin: boolean;
+
+	constructor(
+		private _userService: UserService,
+		private _matchService: MatchService,
+		private _toasterService: ToasterService
+	) {}
+
+	async ngOnInit() {
+		this.isAdmin = this._userService.havePermission("admin");
+		try {
+			this.userMatches = (await this._matchService.get()).filter(
```

Comment:
I'm going to allow this for now, but in the future we should really not fetch hundreds of matches from the server :| I'm assuming that if we make it possible to handle matches for others, we'll have some endpoint to fetch all matches for a user

Reply:
Yeah. I started making an endpoint for fetching only matches for a single user, however, I quickly ran into issues:
It cannot be a getAll, since we cannot pass data with gets, and we have a terrible query param system. It did not make sense to make a POST either, so I settled on the simple solution, since we only have a limited number of matches.